README.md

```diff
@@ -4,36 +4,35 @@
 
 This extension to Firefox provides the ability to work with tabs as "trees".
 
-New tabs opened from the current tab are automatically organized as "children" of the current.
-Such "branches" are easily folded (collapsed) by clicking on down on the arrow shown in the "parent" tab, so you don't need to suffer from too many visible tabs anymore.
+New tabs opened from the current tab are automatically organized as "children" of the current tab.
+Such "branches" are easily folded (collapsed) by clicking on the arrow shown in the "parent" tab, so you no longer need to suffer from too many visible tabs.
 If you want, you can restructure the tree via drag and drop.
 
  * Such a tree of tabs will behave like a visual browsing history for you.
-   For example, if you see a list of search results for a topic, you'll open each search result link in new child tab.
-   For more details you'll also open more descendant tabs from them.
+   For example, if you see a list of search results for a topic, each search result link will open in new child tab.
+   New tabs opened from these "child" tabs will appear as descendants of their originating tabs.
    You'll easily dig and dig deeply, without losing your browsing trail - if you want to go back to the original search result, you just have to switch to the "root" tab.
- * Moreover, you'll treat a tree of tabs just as "grouped tabs" for similar topics.
+ * Moreover, you'll each tree will reflect a group of tabs on a similar topic.
```

Comment:
Missing "treat" ?

Reply:
Ah, thanks!  Actually, the "treat" should be removed, but I missed also removing the "you'll" right before it.  I'll fix it now for you.